StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/CustomSheet/CustomSheetLayoutView.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  CustomSheetLayoutView.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 6/5/23.
+//
+
+import Foundation
+@_spi(STP) import StripeUICore
+import UIKit
+
+/// Reusable view that separates panes into two parts:
+/// 1. A scroll view for content
+/// 2. A footer that is "locked" and does not get affected by scroll view
+///
+/// Purposefully NOT a `UIView` subclass because it should only be used via
+/// `addToView` helper function.
+final class CustomSheetLayoutView {
```

Comment:
This is an "infra" component for sheets. 

It allows to easily set a `contentView` and `footerView`, and it embeds this into an automatically-resizable "sheet" with a "handle" on top